Source/WebCore/platform/graphics/gstreamer/mse/PlaybackPipeline.cpp

```diff
@@ -371,14 +371,6 @@ void PlaybackPipeline::flush(AtomicString trackId)
         return;
     }
 
-    if (!gst_element_send_event(GST_ELEMENT(appsrc), gst_event_new_flush_stop(false))) {
-        GST_WARNING("Failed to send flush-stop event for trackId=%s", trackId.string().utf8().data());
-        return;
-    }
-
-    if (static_cast<guint64>(position) == GST_CLOCK_TIME_NONE || static_cast<guint64>(start) == GST_CLOCK_TIME_NONE)
```

Comment:
It's ok to remove the check for invalid position (it has already been checked in line 352), but why remove the check for invalid start time? I can't think of any use case where a segment with invalid start time would happen... except maybe in reverse playback, but still it's very strange.

Reply:
I wanted to make the patch smaller. So, I decided to remove the check, because I didn't see any usage of 'start' in the code. Let me know if there is a reason to do an early return if start set to GST_CLOCK_TIME_NONE

Reply:
Yes, ok, a segment without start time is unlikely to happen in practice.